src/main/resources/db/migration/V23__flatten_categories_and_audit_benefit_structuring.sql

```
@@ -0,0 +1,337 @@
+    ON version.content_version_id = benefit.content_version_id
+INNER JOIN cards card ON card.card_id = version.card_id
+WHERE card.gorilla_card_id = '2852'
+  AND benefit.title = '편의점'
+  AND target.target_type = 'merchant_category';
+
+INSERT INTO benefit_rule_targets
+    (target_id, rule_id, condition_group, match_mode, target_type,
+     merchant_category_id, merchant_id, target_code, target_name,
+     target_source, target_authority, minimum_place_confidence, created_at, updated_at)
+SELECT UUID(), rule_data.rule_id, seed.condition_group, 'include', 'merchant',
+       NULL, merchant.merchant_id, seed.merchant_name, seed.merchant_name,
```

SQLFluff (4.2.2)

[error] Reuse of column alias 'merchant_name' from line 144.

(AL08)